mod_reforged/hooks/entity/tactical/humans/bounty_hunter.nut

```diff
@@ -39,5 +39,46 @@
 	{
 		__original();
 		::Reforged.Skills.addPerkGroupOfEquippedWeapon(this, 4);
+
+		if (this.getBodyItem() != null)
+		{
+			local armor = this.getBodyItem();
+			local armorAttachment = null;
+
+			if (armor.getConditionMax() < 115)
+				{
+					if (::Math.rand(1, 100) <= ::Reforged.Config.ArmorAttachmentChance.Tier3)
+					{
+						armorAttachment = ::Reforged.ItemTable.ArmorAttachmentNorthern.roll({
+							Apply = function ( _script, _weight )
+							{
+								local conditionModifier = ::ItemTables.ItemInfoByScript[_script].ConditionModifier;
+								if (conditionModifier > 20) return 0.0;
+								return _weight;
+							}
+						})
+					}
+				}
+
+			else if (armor.getConditionMax() >= 115)
+			{
+				if (::Math.rand(1, 100) <= ::Reforged.Config.ArmorAttachmentChance.Tier3)
+				{
+					armorAttachment = ::Reforged.ItemTable.ArmorAttachmentNorthern.roll({
+						Apply = function ( _script, _weight )
+						{
+							local conditionModifier = ::ItemTables.ItemInfoByScript[_script].ConditionModifier;
+							if (conditionModifier > 30) return 0.0;
+							return _weight;
+						}
+					})
+				}
+			}
```

Comment:
This `if/else` block can be simplified massively as follows:
```suggestion
			local conditionModifierCutOff = armor.getConditionMax() < 115 ? 20 : 30;
			if (::Math.rand(1, 100) <= ::Reforged.Config.ArmorAttachmentChance.Tier3)
			{
				armorAttachment = ::Reforged.ItemTable.ArmorAttachmentNorthern.roll({
					Apply = function ( _script, _weight )
					{
						local conditionModifier = ::ItemTables.ItemInfoByScript[_script].ConditionModifier;
						if (conditionModifier > conditionModifierCutOff) return 0.0;
						return _weight;
					}
				})
			}
```

Also the `::Math.rand(1,100 <= ::Reforged.Config.ArmorAttachmentChance.Tier3` condition can be moved up with an `&&` into the `this.getBodyItem() != null` check.